<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="mainLayout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="btn_ImportData">
    <property name="title">
     <string>Import Data</string>
    </property>
   </widget>
   <widget class="QMenu" name="btn_Calender">
    <property name="title">
     <string>Calender</string>
    </property>
   </widget>
   <widget class="QMenu" name="btn_Graphs">
    <property name="title">
     <string>Graphs</string>
    </property>
   </widget>
   <widget class="QMenu" name="btn_Transformations">
    <property name="title">
     <string>Transformations</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="btn_ImportData"/>
   <addaction name="btn_Calender"/>
   <addaction name="btn_Graphs"/>
   <addaction name="btn_Transformations"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
